src/utils/__tests__/nodeUtils.test.ts

```diff
@@ -38,4 +38,25 @@ describe('node util methods', () => {
     const parents = findParents(samplepathway, 'ChemoMedication');
     expect(parents).toEqual(['N-test', 'Surgery']);
   });
+
+  describe('get connectable nodes', () => {
```

Comment:
For these tests - instead of returning true/false can we make them check the return objects are identical (since we can figure out what they should be from the samplePathway)

Reply:
Yes. I'm going to change the second one test to check the return object, but keep the first one as is because it's the only way I can think of to quickly check that every node in the pathway is finding potential transitions as expected.